src/pages/ReservationStatusPage/components/ReservationCard.jsx

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import FilterMatching from "../components/FilterMatching";
+import ProfileBookBtn from "../../../components/ProfileBookBtn/ProfileBookBtn";
+import styled from "styled-components";
+
+const ReservationCard = props => {
+  const { host, guest, court, reservation } = props;
```

Comment:
```suggestion
const ReservationCard =({ host, guest, court, reservation }) => {
```

구조분해할당을 이렇게 하면 더 깔끔하겠네요!